clients/admin-ui/src/features/privacy-requests/hooks/useMutations.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  useApproveRequestMutation,
+  useDenyRequestMutation,
+} from "~/features/privacy-requests/privacy-requests.slice";
+import { PrivacyRequestEntity } from "~/features/privacy-requests/types";
+
+export const useMutations = ({
+  subjectRequest,
+}: {
+  subjectRequest: PrivacyRequestEntity;
+}) => {
+  // The `fixedCacheKey` options allows multiple components to reference the same mutation state for
+  // a subject request.
```

Comment:
The privacy requests table could benefit from even more shared state management -- there are a lot of callbacks passed deep down the tree. This was a handy way to start: https://redux-toolkit.js.org/rtk-query/usage/mutations#shared-mutation-results

Reply:
This is really cool. I didn't know you could do this with RTK